.golangci.yml

```diff
@@ -1,12 +1,13 @@
 linters-settings:
   gocyclo:
-    min-complexity: 40 # for dashboard ReconcileComponent()
+    min-complexity: 30
+  lll:
+    line-length: 180
 
 linters:
   disable-all: true
   enable:
```

Comment:
i am  thinking to take "enable-all:true"  as a follow up issue,
i did a try  but too many need to fix to have this one close,  
it can be a "first-goo" for the new team members to take a look + helping them to go thro code.
how you feel @bartoszmajsak

Reply:
Yeah it can be a snowball exercise :) Making it a follow-up task, assuming it won't be covered in dust, is a good approach :)

Reply:
created https://github.com/opendatahub-io/opendatahub-operator/issues/553